<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QWidget" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{background-color: #13100a; border-style: solid;  border-width: 1px; border-radius: 10px; color: white; font-size: 20px; font-family: 'Arial'; font-weight: 500px;}
QPushButton:hover{background-color: #24211b;}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>281</width>
     <height>292</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="m_OptionsTitle">
      <property name="styleSheet">
       <string notr="true">font-size: 40px; font-weight: 800;</string>
      </property>
      <property name="text">
       <string>Options</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 75 italic 14pt &quot;Cambria&quot;;</string>
      </property>
      <property name="text">
       <string>Languages:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="choiceComboBox">
      <property name="styleSheet">
       <string notr="true">background-color: black; color: red; font: italic 12pt &quot;Cambria&quot;;</string>
      </property>
      <item>
       <property name="text">
        <string>Polski</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>English</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Deutche</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ruski</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: italic 12pt &quot;Cambria&quot;;</string>
        </property>
        <property name="text">
         <string>1. W - up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: italic 12pt &quot;Cambria&quot;;</string>
        </property>
        <property name="text">
         <string>2. D - right</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font: italic 12pt &quot;Cambria&quot;;</string>
        </property>
        <property name="text">
         <string>3. S - down</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">font: italic 12pt &quot;Cambria&quot;;</string>
        </property>
        <property name="text">
         <string>4. A - left</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">font: italic 12pt &quot;Cambria&quot;;</string>
        </property>
        <property name="text">
         <string>5. P - pause</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">font: italic 12pt &quot;Cambria&quot;;</string>
        </property>
        <property name="text">
         <string>6. Space - start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_BackToMenuButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
